src/Sulu/Bundle/WebsiteBundle/Resources/config/analytics.xml

```diff
@@ -3,16 +3,8 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="sulu_website.analytics.repository"
-                 class="Sulu\Bundle\WebsiteBundle\Entity\AnalyticsRepository">
-            <factory service="doctrine.orm.entity_manager" method="getRepository"/>
-
-            <argument type="string">SuluWebsiteBundle:Analytics</argument>
-
-            <call method="setEnvironment">
-                <argument>%kernel.environment%</argument>
-            </call>
-        </service>
+        <service id="sulu_website.analytics.repository" alias="sulu.repository.analytics"/>
+        <service id="Sulu\Bundle\WebsiteBundle\Entity\AnalyticsRepositoryInterface" alias="sulu.repository.analytics"/>
```

Comment:
@nnatter see here

Reply:
would you mind to the `SuluWebsiteExtension` to be consistent with the other bundles. i am afraid that people (like me 😅) will copy the code from the extension class and forget abput the alias in the xml file